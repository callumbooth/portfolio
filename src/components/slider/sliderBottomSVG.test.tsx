import React from "react";
import { render } from "@testing-library/react";
import SliderBottomSVG from "./sliderBottomSVG";

it("Svg rotates to 45 deg correctly", () => {
  const { container } = render(<SliderBottomSVG rotation={45} />);

  expect(container).toMatchSnapshot();
});

it("Svg rotates by 90 deg correctly", () => {
  const { container } = render(<SliderBottomSVG rotation={90} />);

  expect(container).toMatchSnapshot();
});
