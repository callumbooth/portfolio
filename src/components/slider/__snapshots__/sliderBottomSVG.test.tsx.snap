// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg rotates by 90 deg correctly 1`] = `
<div>
  <svg
    viewBox="0 0 1520 1080"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <path
        class="b b1"
        d="M0,0 l919,919 h-218 l-701,-701 Z"
        transform="translate(0 190) rotate(90 459.5 459.5)"
      />
      <line
        class="a l3"
        transform="translate(0 410) rotate(90 350.5 350.5)"
        x2="701"
        y2="701"
      />
      <line
        class="a l4"
        transform="translate(0 190) rotate(90 460.5 460.5)"
        x2="921"
        y2="921"
      />
      <line
        class="a l5"
        transform="translate(1086 669) rotate(90 205.5 205.5)"
        x2="411"
        y2="411"
      />
      <line
        class="a l6"
        transform="translate(1008 873)"
        x2="207"
        y2="207"
      />
    </g>
  </svg>
</div>
`;

exports[`Svg rotates to 45 deg correctly 1`] = `
<div>
  <svg
    viewBox="0 0 1520 1080"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <path
        class="b b1"
        d="M0,0 l919,919 h-218 l-701,-701 Z"
        transform="translate(0 190) rotate(45 459.5 459.5)"
      />
      <line
        class="a l3"
        transform="translate(0 410) rotate(45 350.5 350.5)"
        x2="701"
        y2="701"
      />
      <line
        class="a l4"
        transform="translate(0 190) rotate(45 460.5 460.5)"
        x2="921"
        y2="921"
      />
      <line
        class="a l5"
        transform="translate(1086 669) rotate(45 205.5 205.5)"
        x2="411"
        y2="411"
      />
      <line
        class="a l6"
        transform="translate(1008 873)"
        x2="207"
        y2="207"
      />
    </g>
  </svg>
</div>
`;
